climpred/classes.py

```diff
@@ -91,6 +98,56 @@ def __init__(self, xobj):
     def __repr__(self):
         return _display_metadata(self)
 
+    def smooth(self, smooth_dict='goddard2013'):
+        """Smooth all entries of PredictionEnsemble in the same manner to be
+        able to still calculate prediction skill afterwards.
+
+        Args:
+          xobj (xarray object):
+            decadal prediction ensemble output.
+
+        Attributes:
+            smooth_dict (dict or str): Dictionary to specify the dims to
+                smooth combaticle with `spatial_smoothing_xesmf`,
+                `temporal_smoothing` or `spatial_smoothing_xrcoarsen`.
+                Shortcut for Goddard et al. 2013 recommendations:
+                'goddard2013'
+        """
+        # get proper smoothing function based on smooth args
+        if isinstance(smooth_dict, str):
+            if smooth_dict == 'goddard2013':
+                smooth_fct = smooth_goddard_2013
+                smooth_dict = {'lead': 4}  # default
+            else:
+                raise ValueError(
+                    'Please provide from list of available smoothings: \
+                     ["goddard2013"]'
+                )
+        elif isinstance(smooth_dict, dict):
+            print(smooth_dict, type(smooth_dict))
+            if 'lon' in smooth_dict or 'lat' in smooth_dict:
+                smooth_fct = spatial_smoothing_xesmf
+            elif 'lead' in smooth_dict:
+                smooth_fct = temporal_smoothing
+            elif list(smooth_dict.keys())[0] in list(self.initialized.dims):
```

Comment:
I don't fully understand this logic; maybe a comment

Reply:
wip so far. Needs to change and commented.

Reply:
now changed but its still not very explicit.